### ISSUE TITLE
It may not be clear that clicking on a language is required

### ISSUE THREAD

**Issue by andy5995** [mhwkb/mhwkb.github.io#120]
Will it be clear to visitors that they must select a language to go anywhere on this site?

**Comment by erichoog** [comment]
I agree that it might not be clear to new users that a language might need to be selected but also not 100% sure what a good solution for that would be.  I see there's a paragraph that explains to select one so that might suffice.  We could put the language selector underneath that paragraph possibly as well but not sure it makes sense to have it in two places on the homepage

**Comment by andy5995** [comment]
Thanks for offering some perspective on this, Eric. I have the same problem: not sure what the best solution is yet.

**Comment by KHurleyDesign** [comment]
What if the top navigation was a sticky header and followed the user down the page? That way its always accessible or there was some sort of prompt next to the language selection like "select one".

**Comment by andy5995** [comment]
That sounds like a good solution to me. I can't think of any downsides to it. @erichoog @escottalexander what do you think?

Thanks for the suggestion @KHurleyDesign

**Comment by escottalexander** [comment]
Are we saying that the language selection will follow a user down the page until they select it and then it will leave them alone? I like that idea because it will help a new user realize that there are other languages available.

**Comment by KHurleyDesign** [comment]
I was just suggesting it followed them period, but I think that your idea would be good as well. I am just not sure code wise how that would work.

**Comment by erichoog** [comment]
Is this what you mean @KHurleyDesign?

https://getbootstrap.com/docs/3.3/examples/navbar-fixed-top/

I know I keep going back to bootstrap and we had discussed not using it at one point but the header and footer are both styled with bootstrap right now so we'd just have to add the class `navbar-fixed-top` to the nav element.

Also we'd probably need to add some top padding to the container as the nav may cover some text up at the top depending on the design.  It's in the fixed to top section of the docs https://getbootstrap.com/docs/3.3/components/#navbar

**Comment by KHurleyDesign** [comment]
Yes that is what I was talking about. And I agree with the to padding to the container.

**Comment by erichoog** [comment]
One caution is not to style the container but add a class to the containers where they're at the top of the page.  Because some of the layouts have their own containers like in the footer, if we style all the containers then it might affect the look in the footer.  But I could be wrong!

**Comment by andy5995** [comment]
>I know I keep going back to bootstrap and we had discussed not using it at one point but the header and footer are both styled with bootstrap right now so we'd just have to add the class navbar-fixed-top to the nav element.

That's perfectly fine, unless someone is offering to do it some other way, Eric. :) Thank you for  the suggestion.

**Comment by DarinDev1000** [comment]
Is anyone working on this?
Should I work on it?

**Comment by escottalexander** [comment]
@DarinDev1000 
Be our guest. Are we settled on how to best explain to the user they must click a language? @erichoog @andy5995 @KHurleyDesign 
Perhaps the sticky header should be more descriptive and say "To enter the knowledge base... |Select a language| Thoughts?

**Comment by erichoog** [comment]
I don't believe we ever really decided on that but any ideas are certainly welcome. @escottalexander that sounds like a good idea!

**Comment by DarinDev1000** [comment]
Ok, I will take a look at it.

**Comment by alaajerbi** [comment]
> Ok, I will take a look at it.

@DarinDev1000 If you need any help with this task, let me know please :smile:

**Comment by DarinDev1000** [comment]
Ok, Thanks. I’ll look over it.
